src/test/java/org/scalameter/examples/JBenchExample1.java

```diff
@@ -0,0 +1,145 @@
+package org.scalameter.examples;
+
+import java.util.Random;
+import org.scalameter.*;
+import org.scalameter.japi.*;
+import org.scalameter.japi.annotation.*;
+
+
+@scopes({
+    @scopeCtx(scope = "arrays", context = "root")
+})
+public class JBenchExample1 extends JBench.Quick {
+  private Random random;
+
+  public final Context root = new ContextBuilder()
+      .put("exec.benchRuns", 20)
+      .put("exec.maxWarmupRuns", 10)
+      .build();
+
+  public final Gen<int[]> arrays = JGen.range("size", 10000, 100000, 30000)
+      .map(new Fun1<Integer, int[]>() {
+        @Override
+        public int[] apply(Integer v) {
+          return new int[v];
+        }
+      })
+      .asScala();
```

Comment:
To avoid the `asScala` call, we could either:
- require `JGen` type on generator fields, and have an implicit conversion `JGen -> Gen` in `japi`
- accept both `Gen` and `JGen` as generator type

Thoughts?